Makefile

```diff
@@ -164,7 +164,7 @@ ifndef ignore-not-found
   ignore-not-found = false
 endif
 
-clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=gcr.io/k8s-staging-kueue/kueue:main)
```

Comment:
why was this set to main and not 0.1.1?

Reply:
ditto the other instances below

Reply:
This is just for the release-0.2 branch (I fixed the target).